packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -955,6 +1013,12 @@ class IOSDeviceLogReader extends DeviceLogReader {
   }
 }
 
+enum IOSDeviceLogSource {
+  iosDeploy,
+  idevicesyslog,
+  unifiedLogging,
```

Comment:
what's unifiedLogging? Maybe these should have dartdocs.

Reply:
Oh, does that mean we're grabbing both iosDeploy AND idevicesyslog in case we need to fall back to the latter?

Reply:
No, it's actually like a third way to get device logs lol. I haven't looked into it too deeply but I believe it gets the logs from the VM
https://github.com/flutter/flutter/blob/ff33555b239de4c17e7149db17eb70bcf5dee215/packages/flutter_tools/lib/src/ios/devices.dart#L828-L870

Reply:
:(